cert-manager-multi-arch.md

```diff
@@ -0,0 +1,718 @@
+- [Manage certificates on OpenShift running on IBM Z®, IBM Power® and ARM64 architectures.](#manage-certificates-on-openshift-running-on-ibm-z-ibm-power-and-arm64-architectures)
+  - [Intoduction](#intoduction)
+  - [Multi-architecture container image](#multi-architecture-container-image)
+    - [What is it?](#what-is-it)
+    - [How to create?](#how-to-create)
+    - [How to verify?](#how-to-verify)
+  - [Deploy OpenShift cluster on IBM Power VS](#deploy-openshift-cluster-on-ibm-power-vs)
+    - [Create IBM Power VS workspace](#create-ibm-power-vs-workspace)
+    - [Download ccoctl, installer and oc utilities](#download-ccoctl-installer-and-oc-utilities)
+    - [Create cluster installation manifests](#create-cluster-installation-manifests)
+    - [Identity and access management](#identity-and-access-management)
+    - [Deploy the cluster](#deploy-the-cluster)
+  - [Install cert-manager on IBM Power VS](#install-cert-manager-on-ibm-power-vs)
+    - [Review OLM resources](#review-olm-resources)
+    - [Operator Installation with CLI](#operator-installation-with-cli)
+  - [Certificate Management](#certificate-management)
+    - [Create Self-signed CA Issuer](#create-self-signed-ca-issuer)
+    - [Issue Certificate](#issue-certificate)
+    - [Change ingress controller's default certificate](#change-ingress-controllers-default-certificate)
+    - [Replace the CA bundle certificate](#replace-the-ca-bundle-certificate)
+    - [See it in action](#see-it-in-action)
+  - [Wrap Up](#wrap-up)
+  - [Useful links](#useful-links)
+
+
+# Manage certificates on OpenShift running on IBM Z®, IBM Power® and ARM64 architectures.
+
+
+## Intoduction
+
+Kubernetes thrives on secure communication, but manually managing SSL/TLS certificates is a tangled mess and error-prone process, leading to outages, security vulnerabilities, and operational overhead.
+
+
+The [cert-manager](https://docs.openshift.com/container-platform/4.14/security/cert_manager_operator/index.html) Operator for Red Hat OpenShift provides a secure and efficient solution for TLS certificate management in OpenShift Container Platform clusters, by introducing certificates and certificate issuers as primary resources in the Kubernetes API.  This _**'certificates as a service'**_ model seamlessly integrates with external certificate authorities, automating the entire certificate lifecycle, from provisioning to renewal, ensuring validity and timely updates.
+
+Now, with the latest update, the cert-manager Operator for Red Hat OpenShift `v1.13.0` has undergone an expansion in its scope. Formerly confined to supporting solely on `AMD64` architecture, it now includes extended support for managing certificates on OpenShift across multiple architectures, including IBM Z® (`s390x`), IBM Power® (`ppc64le`), and `ARM64`. 
```

Comment:
just a suggestion, upto you.
```suggestion
Now, with the latest update, the cert-manager Operator for Red Hat OpenShift `v1.13.0` has undergone an expansion in its scope. Formerly confined to `AMD64` architecture, it now includes extended support for managing certificates on OpenShift across multiple architectures, including IBM Z® (`s390x`), IBM Power® (`ppc64le`), and `ARM64`. 
```

Reply:
I think it emphasizes more on the improvement, so we can keep this phrase.